VDA-SBT/contracts/SoulboundNFT.sol

```diff
@@ -36,10 +36,10 @@ contract SoulboundNFT is VDAVerificationContract,
     EnumerableSet.StringSet private _sbtTypes;
 
     /**
-     * @notice Claimed SBT info by users
-     * @dev mapping of User => SBTType => UniqueId => tokenId
+     * @notice Claimed token id for SBTType & UniqueId
+     * @dev mapping SBTType => UniqueId => tokenId
```

Comment:
Update this mapping

Reply:
Let's assum there is SBTType : "twitter" & UniqueID : "test123".
DID A claimed for this.
Can another DID claim a sbt for this SBTType & UniequeID?